src/explorer.api/Models/DiffixApiClient.cs

```diff
@@ -70,49 +71,108 @@ public enum DiffixType
         IEnumerable<Table> Tables { get; set; }
     }
 
-    public class QueryResult<RowType>
+    /** <summary>
+         Helper type representing the JSON response from a request to /api/queries/{query_id}.
+     
+         Example response: 
+        <code>
+        {
+            &quot;query&quot;: {
```

Comment:
Oh man this is nasty to read in source code form... Isn't there a way to make it readable here too? At the moment it provides more noise than value?

Reply:
Sadly I think the &quot;ugliness&quot; is necessary if we want the documentation to be correctly generated. Alternatively, for more extensive docs we could have documentation in separate .xml files and then use an <include> tag to pull in the file contents. This has the downside that the documentation isn't visible while developing so would be best used if we want some kind of user-facing docs. 

I agree this is an extreme case, I don't mind removing it and putting it somewhere else for future reference (the wiki?). 

Btw. I have simply been using the editor to fold the documentation section away.

Reply:
Ok, folding away is OK then I suppose